src/logo/openmrs-logo-full.svg

```diff
@@ -0,0 +1,32 @@
+<svg viewBox="0 0 380 119" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path fill-rule="evenodd" clip-rule="evenodd" d="M40.29 40.3278C45.3275 35.2904 52.2878 32.1739 59.9776 32.1739C67.6466 32.1739 74.5911 35.2765 79.6245 40.29L79.6454 21.7502C73.7513 18.7143 67.0622 17 59.9776 17C52.8891 17 46.1652 18.9309 40.2692 21.9679L40.29 40.3278Z" fill="var(--logo-red)"/>
```

Comment:
Notice the `var(--logo-red)`. I modified all the svgs to use css variables for their colors, so it would be easy to swap them out for the mono color versions of them.